components/Form/index.js

```diff
@@ -0,0 +1,150 @@
+import { styled } from "styled-components";
```

Comment:
I wonder, isn't `styled` a **default export?** I was under the impression it's not a named one.